docs.json

```diff
@@ -2260,15 +2461,227 @@
+    },
+    {
+      "source": "/guides/integrations/metaflow",
+      "destination": "/models/integrations/metaflow"
+    },
+    {
+      "source": "/guides/integrations/mmengine",
+      "destination": "/models/integrations/mmengine"
+    },
+    {
+      "source": "/guides/integrations/mmf",
+      "destination": "/models/integrations/mmf"
+    },
+    {
+      "source": "/guides/integrations/nim",
+      "destination": "/models/integrations/nim"
+    },
+    {
+      "source": "/guides/integrations/openai-api",
+      "destination": "/models/integrations/openai-api"
+    },
+    {
+      "source": "/guides/integrations/openai-fine-tuning",
+      "destination": "/models/integrations/openai-fine-tuning"
+    },
+    {
+      "source": "/guides/integrations/openai-gym",
+      "destination": "/models/integrations/openai-gym"
+    },
+    {
+      "source": "/guides/integrations/paddledetection",
+      "destination": "/models/integrations/paddledetection"
+    },
+    {
+      "source": "/guides/integrations/paddleocr",
+      "destination": "/models/integrations/paddleocr"
+    },
+    {
+      "source": "/guides/integrations/prodigy",
+      "destination": "/models/integrations/prodigy"
+    },
+    {
+      "source": "/guides/integrations/pytorch-geometric",
+      "destination": "/models/integrations/pytorch-geometric"
+    },
+    {
+      "source": "/guides/integrations/pytorch",
+      "destination": "/models/integrations/pytorch"
+    },
+    {
+      "source": "/guides/integrations/ray-tune",
+      "destination": "/models/integrations/ray-tune"
+    },
+    {
+      "source": "/guides/integrations/sagemaker",
+      "destination": "/models/integrations/sagemaker"
+    },
+    {
+      "source": "/guides/integrations/scikit",
+      "destination": "/models/integrations/scikit"
+    },
+    {
+      "source": "/guides/integrations/simpletransformers",
+      "destination": "/models/integrations/simpletransformers"
+    },
+    {
+      "source": "/guides/integrations/skorch",
+      "destination": "/models/integrations/skorch"
+    },
+    {
+      "source": "/guides/integrations/spacy",
+      "destination": "/models/integrations/spacy"
+    },
+    {
+      "source": "/guides/integrations/stable-baselines-3",
+      "destination": "/models/integrations/stable-baselines-3"
+    },
+    {
+      "source": "/guides/integrations/tensorboard",
+      "destination": "/models/integrations/tensorboard"
+    },
+    {
+      "source": "/guides/integrations/tensorflow",
+      "destination": "/models/integrations/tensorflow"
+    },
+    {
+      "source": "/guides/integrations/torchtune",
+      "destination": "/models/integrations/torchtune"
+    },
+    {
+      "source": "/guides/integrations/ultralytics",
+      "destination": "/models/integrations/ultralytics"
+    },
+    {
+      "source": "/guides/integrations/w-and-b-for-julia",
+      "destination": "/models/integrations/w-and-b-for-julia"
```

Comment:
I think all of these file-specific `/guides/integrations` redirects are redundant since you have a `:slug*` redirect that should catch all of them down below